### PR TITLE
Implement `verify-remote` for checking GitHub release signatures

### LINKED ISSUE #5: Implement periodic check of signatures + checksums on GitHub releases
... in order to catch things like this early: https://github.com/rnpgp/rnpgp.org/pull/44#issuecomment-898758086

### PR DESCRIPTION
**Pull request by ribose-jeffreylau**
Fixes #5 

This PR also sets up a signature check every Wednesday and Sunday.

### CODE REVIEW

**Comment by ribose-jeffreylau** [comment]
It did successfully catch a missed signature update! (fixed now)

See: https://github.com/rnpgp/release-sign/runs/3350002381?check_suite_focus=true#step:3:18

File: spec/tests.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+
+KEY_1_FPR=A845A5BD622556E89D7763B5EB06D1696BEC4C90
+
+test_package_signature 0.9.1 "${KEY_1_FPR}"
```
Comment (ni4) [inline spec/tests.sh]: What about using `git ls-remote --tags https://github.com/rnpgp/rnp` or something like this here, so future tags will be picked as well?

File: rel-sign.sh [hunk 2]
```diff
@@ -325,6 +360,22 @@ sign() {
 	infop "Signatures are stored in files %s and %s.\n" "v${VERSION}.tar.gz.asc" "v${VERSION}.zip.asc";
 }
 
+verify-remote() {
+	# Validate hashes first
```
Comment (ni4) [inline rel-sign.sh]: Is `validate hashes first` for future? As doesn't see hash verification yet :)

**Review by ni4 — APPROVED** [review]
LGTM, but please see comments as well.

File: rel-sign.sh (hunk 2 above)
Comment (ribose-jeffreylau) [reply rel-sign.sh]: Good catch, thanks! :p

**Pull request merged**